### PR TITLE
Lev 194

### PR DESCRIPTION
**Pull request by aarongill1**
Parameterised proxies for pipeline to use them.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM node:20-alpine3.18 as base
+FROM node:20-alpine3.16 as base
```
Comment (mehar-chauhan) [inline Dockerfile:1]: Is this downgrade deliberate?

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-ENV HTTP_PROXY='http://proxy.local:8080'
-ENV HTTPS_PROXY='http://proxy.local:8080'
+ARG HTTP_PROXY=''
+ARG HTTPS_PROXY=''
```
Comment (mehar-chauhan) [inline Dockerfile:4]: Presuming you can't do aware with these fields given that you're passing in `--builg-args` in the CLI?

File: Dockerfile-e2e [hunk 3]
```diff
@@ -2,16 +2,15 @@ FROM 113355358853.dkr.ecr.eu-west-1.amazonaws.com/ho/hmpo/lev/fro/cypress-testin
...
+ARG HTTPS_PROXY=''
 
 WORKDIR /app
 COPY ./ /app
 
 RUN npm ci -D
 
-ENV HTTP_PROXY=''
-ENV HTTPS_PROXY=''
+ARG NO_PROXY=''
```
Comment (mehar-chauhan) [inline Dockerfile-e2e:13]: If this needs to be set, then just a very minor thing, but could you keep the `ARG`s next to each-other?

**Review by mehar-chauhan — APPROVED** [review]
I've approved, but feel free to apply any cosmetic changes suggested in the comments

**Comment by mehar-chauhan** [comment]
@aarongill1 did you want to add that `logs/` folder to this `.gitignore` file similar to what you had added in https://github.com/UKHomeOffice/lev-api/pull/145 ?

File: Dockerfile-e2e (hunk 3 above)
Comment (mehar-chauhan) [reply Dockerfile-e2e:13]: Interestingly this isn't set in https://github.com/UKHomeOffice/lev-api/pull/145, is this required?

File: Dockerfile (hunk 1 above)
Comment (aarongill1) [reply Dockerfile:1]: Yes - there is a bug introduced in 3.17 that causes errors in docker-compose. 3.17/3.18 are seemingly less stable and are having frequent updates, so downgraded at least for now.

File: Dockerfile-e2e (hunk 3 above)
Comment (aarongill1) [reply Dockerfile-e2e]: I have removed

File: Dockerfile (hunk 2 above)
Comment (aarongill1) [reply Dockerfile]: I thought we needed them for the docker build but seemingly nor, so I have removed.

**Pull request merged**